packages/3id-did-resolver/src/index.ts

```diff
@@ -10,57 +19,43 @@ interface Ceramic {
   createDocument(type: string, content: any, opts: any): Promise<Doctype>;
 }
 
-interface ResolverRegistry {
-  [index: string]: DIDResolver;
-}
-
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export function wrapDocument(content: any, did: string): DIDDocument {
   if (!(content && content.publicKeys)) throw new Error('Not a valid 3ID')
   const startDoc: DIDDocument = {
-    '@context': 'https://w3id.org/did/v1',
     id: did,
-    publicKey: [],
+    verificationMethod: [],
     authentication: [],
-    keyAgreement: []
+    keyAgreement: [],
+    publicKey: []
   }
-  const doc = Object.entries(content.publicKeys as string[]).reduce((diddoc, [keyName, keyValue]) => {
+  return Object.entries(content.publicKeys as string[]).reduce((diddoc, [keyName, keyValue]) => {
     const keyBuf = u8a.fromString(keyValue.slice(1), 'base58btc')
-    // remove multicodec varint
-    const publicKeyBase58 =  u8a.toString(keyBuf.slice(2), 'base58btc')
+    const entry: VerificationMethod = {
```

Comment:
I am not sure if _all_ of the keys should get here. x25519 is meaningless for verification.

Does not hurt AFAIK, but really I am not sure if it should get here.